projects/admin.py

```diff
@@ -1,5 +1,6 @@
 from django.contrib import admin
 
-from .models import Project
+from .models import Project, Client
```

Comment:
nit and personal preference: how do you feel about alphabetizing these imports (as well as on lines 5-6)? I tend to like the consistency of that pattern, but feel free to ignore :)